benchmarks/ekobox/benchmark_evol_pdf.py

```diff
@@ -91,8 +91,14 @@ def benchmark_evolve_more_members(tmp_path, cd, lhapdf_path):
             ev_p.evolve_pdfs(pdfs, theory, op, install=True, name="Debug")
         # ev_pdfs
         _ = lhapdf.mkPDFs("Debug")
```

Comment:
Maybe now you don't need any longer this line

Reply:
I would keep it because it is testing if lhapdf is able to load the complete set

Reply:
Is it doing something different than loading the info and the individual replicas?

Reply:
If so, don't discard the return value, but test what you expect it to be.

Reply:
No it is only loading the info and all the replicas and constructing the lhapdf object. Maybe I can just check if the number of final replicas is the same of the initial number

Reply:
Have a look if now it is ok